frontend/src/app/settings/core/tools-settings/tool-details/tool-deletion-dialog/tool-deletion-dialog.component.html

```diff
@@ -3,8 +3,8 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-<h1 mat-dialog-title>Remove Tool</h1>
-<div mat-dialog-content>
+<h1>Remove Tool</h1>
+<div>
```

Comment:
```suggestion
<div class="dialog">
```
Similar to other dialogs, this dialogs misses some padding which should be solved by adding the `dialog` class to the div here.